src/Diagnostics.CompilerHost/appsettings.json

```diff
@@ -11,5 +11,10 @@
         "Default": ""
       }
     }
+  },
+  "AzureAd": {
+    "AADAuthority": "https://login.microsoftonline.com/microsoft.onmicrosoft.com",
+    "ClientId": "cda572d8-1f3e-4119-9a11-0e4ceb2fb3e1", // Azure AAd App id of CompilerHost
```

Comment:
lets remove the values of AppIds from AppSettings and keep this file clean.
Hopefully we would move to keyvault after Craig's work.

Reply:
Should we remove in this PR itself? Because once its merged, it wouldn't work. I thought of maintaining appsettings.Development.json, but that would be redundant again.

Reply:
I think using Development.JSON is a better option which you are not going to check-in. Keeping this file just as a schema for config should be good.

Reply:
Anyhow, it would be temporary as we should rely on Craig's work for keyvault